draft-ietf-taps-interface.md

```diff
@@ -2244,28 +2244,28 @@ This property specifies the desired network treatment for traffic sent by the
 application and the tradeoffs the application is prepared to make in path and
 protocol selection to receive that desired treatment. When the capacity profile
 is set to a value other than Default, the transport system should select paths
-and profiles to optimize for the capacity profile specified. The following
-values are valid for the Capacity Profile:
+and protocols to optimize for the capacity profile specified. How this is realized
+is implementation-specific. The Capacity Profile may also be used
+to set priorities on the wire for Protocol Stacks supporting prioritization. 
+Recommendations for use with DSCP are provided below for each profile; note that
+when a Connection is multiplexed, the guidelines in Section 6 of {{?RFC7657}} apply.
+  
+The following values are valid for the Capacity Profile:
 
   Default:
-  : The application makes no representation about its expected capacity
-  profile. No special optimizations of the tradeoff between delay, delay
-  variation, and bandwidth efficiency should be made when selecting and
-  configuring transport protocol stacks. Transport system implementations that
-  map the requested capacity profile onto per-connection DSCP signaling without
-  multiplexing SHOULD assign the DSCP Default Forwarding {{?RFC2474}} PHB; when
-  the Connection is multiplexed, the guidelines in Section 6 of {{?RFC7657}}
-  apply.
+  : The application provides no information about its expected capacity
+  profile. Transport system implementations that
+  map the requested capacity profile onto per-connection DSCP signaling 
+  SHOULD assign the DSCP Default Forwarding {{?RFC2474}} PHB.
```

Comment:
I'm fine with all other changes, but I wonder why you removed "when the Connection is multiplexed, the guidelines in Section 6 of {{?RFC7657}} apply." everywhere?

One could see this as an implementation detail, but then this part gives some hints about the expected behavior regarding DSCP values, if an implementation should choose to adapt the DSCP based on the Capacity Profile. These chosen DSCP values will be different in case of multiplexing, as Section 6 of RFC 7657 explains. So I would prefer to see this sentence stay, everywhere.

Reply:
It has not been removed, just moved from the individual items to the top, as it applies to all classes.